Issue-Embeddings/flask_app/inference.py

```diff
@@ -24,15 +31,18 @@ def __init__(self,
         self.learn.model.eval()  # turn off dropout, etc. only need to do this after loading model.
         self.encoder = self.learn.model[0]
         self.pad_idx = self.learn.data.pad_idx
+        self.mtokenizer = [x.tokenizer for x in self.learn.data.processor if type(x) == TokenizeProcessor][0]
```

Comment:
nit:  What does "m" in tokenizer stand for. We generally eschew abbreviations
http://google.github.io/styleguide/pyguide.html#316-naming

Reply:
👍